libs/src/services/ethWeb3Manager/index.js

```diff
@@ -81,24 +85,25 @@ class EthWeb3Manager {
     }
 
     let gasPrice = parseInt(await this.web3.eth.getGasPrice())
-    return contractMethod.send({ from: this.ownerWallet, gas: gasAmount, gasPrice: gasPrice })
+    return contractMethod.send({ from: this.ownerWallet, gas: gasLimit, gasPrice: gasPrice })
   }
 
   async relayTransaction (
     contractMethod,
     contractAddress,
     ownerWallet,
-    txGasLimit = DEFAULT_GAS_AMOUNT,
-    txRetries = 5
+    txRetries = 5,
```

Comment:
why flip this order?

Reply:
just based on usage. more places do custom retries (e.g. rate limits for $AUDIO sends) and no where is overriding txGasLimit as of yet with this PR